File: Lab1/circle.cpp
#include <iostream>
using namespace std;

void getRadius(int &radius)
{
    cout << "Enter radius: ";
    cin >> radius;
}

void calculateArea(int radius)
{
    double area = 3.14 * radius * radius;
    cout << "Area of circle is " << area << endl;
}

void calculateCircumference(int radius)
{
    double circumference = 2 * 3.14 * radius;
    cout << "Circumference of circle is " << circumference << endl;
}

int main()
{
    int radius;

    // Function calls
    getRadius(radius);
    calculateArea(radius);
    calculateCircumference(radius);

    return 0;
}

File: Lab11/shapes/Circle.java
public class Circle extends Shape{
    private double radius;

    public Circle(double radius){
        this.radius = radius;
    }

    // Getters
    public double getRadius(){
        return this.radius;
    }

    @Override
    public double calculateArea(){
        return Math.PI * this.radius * this.radius;
    }

    @Override
    public double calculatePerimeter(){
        return 2 * Math.PI * this.radius;
    }

    public void display(){
        System.out.println("Circle: ");
        super.display();
    }

}
